app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -29,5 +31,19 @@ def destroy
       form.destroy
       render json: form
     end
+
+    private
+
+    def ensure_uuid
+      # There have been several errors where `@current_user.uuid` is being coerced to `nil`
+      # by activerecord. This checks the the `uuid` against the same regex and logs an error
+      # if we see a 'malformed' id.
+      uuid_format = ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Uuid::ACCEPTABLE_UUID
+      # We should always have a uuid on @current_user. If this fails, that's another issue
+      unless @current_user.uuid.to_s[uuid_format, 0]
```

Comment:
Just copying the AR code verbatim so I know I'm getting exactly the same behavior